packages/formstr-app/src/index.css

```diff
@@ -13,3 +13,17 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.ant-btn-primary {
```

Comment:
Instead of this we can use the theme provider for antd

Advantage of using that is that the themeing is typed and versioned

For example, if we upgrade antd and they change the stylings we will never know why our styles are not applied

Using the theme provider gives us the advantage of typings. If they change the classes/interface we would just need to make sure that we pass in the types correctly to the theme provoder

Reply:
There should be a way to override the button through config provider

Reply:
the theme provider doesn't have a background image token, which makes it impossible to do a gradient button without overriding the css, this was a last resort option